src/main/kotlin/ink/abb/pogo/scraper/controllers/BotController.kt

```diff
@@ -106,32 +111,32 @@ class BotController {
     @RequestMapping(value = "/bot/{name}/pokemons", method = arrayOf(RequestMethod.GET))
     fun listPokemons(@PathVariable name: String): List<PokemonData> {
 
-        service.getBotContext(name).api.inventories.updateInventories(true)
+        //service.getBotContext(name).api.inventories.updateInventories(true)
 
-        val data = service.getBotContext(name).api.inventories.pokebank.pokemons
-        val pokemons = mutableListOf<PokemonData>()
-        for (pokemon in data) {
-            pokemons.add(PokemonData().buildFromPokemon(pokemon))
-        }
-
-        return pokemons
+        return service.getBotContext(name).api.inventory.eggs.map { PokemonData().buildFromPokemon(it.value) }
```

Comment:
`Request processing failed; nested exception is java.lang.IllegalStateException: PokemonMetaRegistry.getMeta(this.getPokemonId()) must not be null`

You try to read Pokemon data from the inventory.eggs (prolly a c/p mistake?)